images/tester/release.sh

```diff
@@ -0,0 +1,5 @@
+set -e
+
+docker build -t fnproject/tester:latest .
```

Comment:
Might want to name this something more descriptive. I can see someone deleting this image since it's sounds kind of temporary.

Reply:
How about fn-test-utils?